cmd/ttn-lw-stack/commands/start.go

```diff
@@ -269,6 +289,12 @@ var startCommand = &cobra.Command{
 					Redis: redis.New(config.Redis.WithNamespace("as", "io", "webhooks")),
 				}
 			}
+			if cfg := config.AS.Webhooks.Templates; cfg.Transport == nil {
+				cfg.Transport = transport
+			}
+			if cfg := config.AS.Interop.InteropClient; cfg.Transport == nil {
+				cfg.Transport = transport
+			}
```

Comment:
I think it's quite error prone to set transport settings here.

Can we were this in the component (context)?